### PR TITLE
Bugfix: Grafana spelled "Graphana"

### PR DESCRIPTION
**Pull request by dbarnesbrownNR**
No additional comments.

### CODE REVIEW

File: src/content/docs/integrations/grafana-integrations/set-configure/configure-new-relic-prometheus-data-source-grafana.mdx [hunk 1]
```diff
@@ -43,7 +43,7 @@ Follow these steps to add New Relic as a Prometheus data source for Grafana.
 8. Click **Save & Test**.
 
 <Callout variant="tip">
-  If your line graphs appear as a series of dots and not as connected lines, you can change in Graphana's **[Graph panel](https://grafana.com/docs/grafana/latest/panels/visualizations/graph-panel/)** and selecting **Stacking and null value > connected**.
+  If your line graphs appear as a series of dots and not as connected lines, you can change in Grafana's **[Graph panel](https://grafana.com/docs/grafana/latest/panels/visualizations/graph-panel/)** and selecting **Stacking and null value > connected**.
```
Comment (austin-schaefer) [inline src/content/docs/integrations/grafana-integrations/set-configure/configure-new-relic-prometheus-data-source-grafana.mdx:46]: The change itself looks fine, but the sentence it's contained in doesn't really read for me. Bolded part:

> If your line graphs appear as a series of dots and not as connected lines, **you can change in** Grafana's graph panel...

**Review by austin-schaefer — APPROVED** [review]
Added one requested tweak @dbarnesbrownNR . 

---

I have another suggestion which is not a blocker but I'd love if we could address it. This doc opens with a LOT of scary important callouts. For such an important path, I'd love to tighten up this journey a bit. Can we move some of these into running text and procedure steps? 

image

Some thoughts:

* Perhaps move the 6.7 one into the intro paragraph as a parenthetical or a few word reference?
* Can we move the "put it in an X-Query-Key" into just a clearer step 7, or delete it entirely?

File: src/content/docs/integrations/grafana-integrations/set-configure/configure-new-relic-prometheus-data-source-grafana.mdx (hunk 1 above)
Comment (dbarnesbrownNR) [reply src/content/docs/integrations/grafana-integrations/set-configure/configure-new-relic-prometheus-data-source-grafana.mdx]: Ah, weird, yeah should have looked at that closer. Did a second PR to fix.

**Comment by dbarnesbrownNR** [comment]
@austin-schaefer agree on all the callouts, that was already on my list for this page and I added it to the ticket just now.

**Pull request merged**